lib/find.js

```diff
@@ -11,6 +11,9 @@ const {getTimezoneAtSea, oceanZones} = require('./oceanUtils')
 
 let featureCache
 
+const featureIndex = require('../data/unindex.json')
+const featureFile = fs.readFileSync('./data/geo.dat')
```

Comment:
consider just opening the file and doing reads of the binary data as needed instead of reading all 24MB into memory?